test/components/scroll-spy.spec.coffee

```diff
@@ -0,0 +1,61 @@
+{React, Testing, expect, _, sinon} = require './helpers/component-testing'
+
+ScrollSpy = require '../../src/components/scroll-spy'
+
+
+NestedComponent = React.createClass
+  render: -> React.createElement('div')
+
+RootComponent = React.createClass
+  render: ->
+
+    React.createElement('div', {},
+      React.createElement(ScrollSpy, @props,
+        React.createElement(NestedComponent, onScreenElements:[])
+      )
+    )
```

Comment:
no, I don't think so. It's a spec file, and all the other's are `.coffee`.  This one sticks out though because it's got a bit of the `React.createElement` in it